server/report_builder/src/build.rs

```diff
@@ -260,3 +260,10 @@ pub fn build(args: BuildArgs) -> anyhow::Result<()> {
 
     Ok(())
 }
+
+pub fn build_report_definition(args: BuildArgs) -> anyhow::Result<ReportDefinition> {
+    let project_dir = Path::new(&args.dir);
+    let files = find_project_files(project_dir)?;
+    let definition = make_report(&args, files)?;
+    Ok(definition)
+}
```

Comment:
New function basically duplicate of 'build', but returns the struct rather than saving as json in dir

Reply:
Would be nice to get the one the writes the JSON to call this one, then save the file.
Just means we don't have two code paths that can get out of sync.